### DISCUSSION TITLE
Documentation edits

### DISCUSSION THREAD

**Discussion by bradh** [ngageoint/tiff-java#37] [General]
I see #32 has been closed. Can I ask if there is a specific problem with that PR, or a more general conceptual issue with the kind of change? I accept that sometimes it's just churn without enough value, but I  am trying to learn the project norms and its hard to infer from so few changes.

**Comment by bosborn** [comment]
The legal ordering and information was set up on early projects by NGA lawyers.  So we just maintain that part of it.

From Markdown lint:
`Not everyone writes markdown in the same way, and there are multiple flavors and styles, each of which are valid. While markdownlint's default settings will result in markdown files that reflect the author's preferred markdown authoring preferences, your project may have different guidelines.`

The README style currently matches all our other projects.  Changing to adapt one project to a different style did not seem worth the value.

**Reply by bradh** [reply]
OK.
